modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/data/provider/ContentDashboardDataProvider.java

```diff
@@ -168,7 +168,10 @@ private AssetVocabularyMetric _getAssetVocabularyMetric(
 		TermsAggregation termsAggregation = _getTermsAggregation(
 			assetVocabulary, assetCategoryTitlesMap.keySet(), "categories");
 
-		termsAggregation.addChildAggregation(childTermsAggregation);
+		termsAggregation.addChildAggregation(
+			_getTermsAggregation(
```

Comment:
Could this change has an impact in the performance of the Audit Graph?

Reply:
No, the aggregation was already being calculated with the latest performance tests. I just take advantage of that data ;)